lib/IRGen/IRGenDebugInfo.cpp

```diff
@@ -1864,7 +1864,7 @@ void IRGenDebugInfoImpl::addFailureMessageToCurrentLoc(IRBuilder &Builder,
   FuncName += failureMsg;
 
   llvm::DISubprogram *TrapSP = DBuilder.createFunction(
-     MainModule, StringRef(), FuncName, TrapLoc->getFile(), 0, DIFnTy, 0,
+     MainModule, FuncName, StringRef(), TrapLoc->getFile(), 0, DIFnTy, 0,
      llvm::DINode::FlagArtificial, llvm::DISubprogram::SPFlagDefinition,
```

Comment:
Can you please also add a testcase to test/DebugInfo ?

Reply:
I updated the original tests (`test/DebugInfo/linetable-codeview.swift` & `test/IRGen/condfail_message.swift') to reflect my changes.

Should I add another test ?

Reply:
No, that is fine! For some reason I didn't see this earlier.